Resources/LogBackends/MessageQueueBackend.py

```diff
@@ -26,23 +27,29 @@ def __init__(self):
     """
     Initialization of the MessageQueueBackend
     """
-    super(MessageQueueBackend, self).__init__(None, JsonFormatter)
+    super(MessageQueueBackend, self).__init__(None, libJsonFormatter, level='verbose')
```

Comment:
I can change it to `debug` if you want.
I set it to `verbose` by default as the `MessageQueueBackend` is mainly used to centralize logs, to avoid putting too much (`debug`) logs in it.
In any case, this can be changed in the configuration.
What would you prefer?

Reply:
that's fine, it was just a question. Maybe to make it clearer, declare a constant `DEFAULT_MQ_LEVEL='verbose'` and use that. It is more explicit.